src/main/java/core/basesyntax/report/ReportMaker.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.report;
+
+import core.basesyntax.model.Fruit;
+
+import java.io.IOException;
+import java.util.List;
+
+public interface ReportMaker {
```

Comment:
I think we can even remove this service and move the logic of creating the report into `storageService`